test/e2e/kubectl/portforward.go

```diff
@@ -202,14 +207,6 @@ func doTestConnectSendDisconnect(bindAddress string, f *framework.Framework) {
 	if err := f.WaitForPodReady(pod.Name); err != nil {
 		framework.Failf("Pod did not start running: %v", err)
 	}
-	defer func() {
-		logs, err := framework.GetPodLogs(f.ClientSet, f.Namespace.Name, pod.Name, "portforwardtester")
-		if err != nil {
-			framework.Logf("Error getting pod log: %v", err)
```

Comment:
This (and similar code in other places) doesn't make much sense anymore, because in case of failure Gomega will print logs we are trying to match.